README.md

```diff
@@ -24,11 +24,11 @@
   <a href="#need-help"><strong>Need help?</strong></a>
 </p>
 
-Timeplus Proton is a streaming SQL engine, a fast and lightweight alternative to ksqlDB or Apache Flink, 🚀 powered by ClickHouse. It enables developers to solve streaming data processing, routing and analytics challenges from Apache Kafka, Redpanda and more sources, and send aggregated data to the downstream systems. Timeplus Proton is the core engine of [Timeplus Enterprise](https://timeplus.com).
+Timeplus Proton is a Stream Processing/Database, a fast and lightweight alternative to ksqlDB or Apache Flink, 🚀 powered by ClickHouse. It enables developers to solve streaming data processing, multi-stream JOINs, sophisticated incremental Materialized Views, routing and analytics challenges from Apache Kafka, Redpanda and more sources, and send aggregated data to the downstream streaming or database systems. Timeplus Proton is the core engine of [Timeplus Enterprise](https://timeplus.com).
```

Comment:
Using capitalized terms is not ideal. How about `Timeplus Proton is a streaming processor/database..`. Also change to `materialized views`.